### PR TITLE
Refactor descriptions

### PR DESCRIPTION
**Pull request by CalvinScott**
#### Overview
It would be nice to clean up the description fields off the field objects.

Make all the other resource use `addDescription()` like [parcels](https://github.com/Retsly/retsly-schemas/blob/master/lib/assessment.js) does.

### CODE REVIEW

**Comment by CalvinScott** [comment]
@CMClay @thalesfsp Please review when free

File: lib/geo.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
-var desc = require('./geo-desc');
+var descriptions = require('./geo-desc');
+var addDescriptions = require('./utils').addDescriptions;
 
-module.exports = {
-  type: {type:'string', default:'Point', desc: desc.type},
-  coordinates: {type:'array', subtype:'number', desc: desc.coordinates}
+var geoSchema = {
+  type: {type:'string', default: 'Point'},
```
Comment (thalesfsp) [inline lib/geo.js]: Added space between default  and 'Point' but forgot between all others? ;)

Comment (thalesfsp) [inline lib/geo.js]: Remember: "consistency is everything"

consistency

**Comment by CalvinScott** [comment]
Updating the wercker file as well per https://github.com/Retsly/api/issues/1515

**Comment by CMClay** [comment]
Nice work @CalvinScott! :+1:
Once you get the thumbs, you can probably:
- merge master
- release a new minor version
- not bother deploying anything

**Comment by thalesfsp** [comment]
:+1:

**Pull request merged**